src/perl/lib/wtsi_clarity/epp/reports/14mg_sample_qc_report.pm

```diff
@@ -188,14 +188,28 @@ sub _get_dna_amount_library_prep {
 
   my $cherrypick_stamping_process_doc = $self->fetch_and_parse($artifact_doc->findvalue($ARTIFACT_PARENT_PROCESS_URI));
 
-  my $volume = $cherrypick_stamping_process_doc->findvalue($UDF_FIELD_REQUIRED_VOLUME);
+  my $volume_required = $cherrypick_stamping_process_doc->findvalue($UDF_FIELD_REQUIRED_VOLUME);
 
-  my $concentration = $cherrypick_stamping_process_doc->findvalue($UDF_FIELD_REQUIRED_CONCENTRATION);
+  my $concentration_required = $cherrypick_stamping_process_doc->findvalue($UDF_FIELD_REQUIRED_CONCENTRATION);
 
-  if ( $concentration eq q{} or $volume eq q{}) {
-    croak "The volume or concentration value is not defined on the sample: $sample_limsid";
+  if ( $concentration_required eq q{} or $volume_required eq q{}) {
+    croak "The volume required or concentration required value is not defined on the sample: $sample_limsid";
```

Comment:
Your english is better ;), but isn't the message better like this?:
The value of the required volume or concentration is not defined on the sample"